api/v1/kubtest.yaml

```diff
@@ -268,9 +265,7 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/ExecutionSummary'
+                $ref: '#/components/schemas/ExecutionsResult'
```

Comment:
@hamzahsn you will need to handle this ExecutionsResult object - which in turn contains an array of ExecutionSummary objects